newIDE/app/src/EventsFunctionsExtensionsLoader/CodeWriters/LocalEventsFunctionCodeWriter.js

```diff
@@ -16,14 +16,14 @@ export const makeLocalEventsFunctionCodeWriter = ({
   onWriteFile,
 }: EventsFunctionCodeWriterCallbacks): EventsFunctionCodeWriter => {
   // The generated code for extensions will be stored in a temporary directory
-  var outputDirTemp ;
+  const outputDirTemp = { value: "" };
```

Comment:
Unfortunately this is making the code just more complex. Read again what I recommend :)

Reply:
Yeah I saw it too late ! I'm sorry
I am starting to doubt of my capacities about resolving this issue